[Bug]: Remove deprecated `pkg_resources` API usage

### Version Number

1.4.13

### Describe the Bug

The `pkg_resources` API has been deprecated for some time and it's usage in `arrapi` produces `DeprecationWarning` messages. It's usage should be removed if possible:

```
.tox/py311/lib/python3.11/site-packages/arrapi/__init__.py:1: in <module>
    import pkg_resources
.tox/py311/lib/python3.11/site-packages/pkg_resources/__init__.py:89: in <module>
    warnings.warn(
E   DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
```

--- a/arrapi/__init__.py
+++ b/arrapi/__init__.py
@@ -1,4 +1,4 @@
-import pkg_resources
+from importlib.metadata import version
 
 from .exceptions import ArrException, ConnectionFailure, Excluded, Exists, Invalid, NotFound, Unauthorized
 from .objs.simple import MetadataProfile, RemotePathMapping, RootFolder, UnmappedFolder, Season
@@ -9,8 +9,8 @@ from .apis.lidarr import LidarrAPI
 from .apis.readarr import ReadarrAPI
 
 try:
-    __version__ = pkg_resources.get_distribution("arrapi").version
-except pkg_resources.DistributionNotFound:
+    __version__ =  version("arrapi")
+except importlib.metadata.PackageNotFoundError::
     __version__ = ""
 __author__ = "Nathan Taggart"
 __credits__ = "meisnate12"

--- a/arrapi/__init__.py
+++ b/arrapi/__init__.py
@@ -10,7 +10,7 @@ from .apis.readarr import ReadarrAPI
 
 try:
     __version__ = version("arrapi")
-except importlib.metadata.PackageNotFoundError::
+except importlib.metadata.PackageNotFoundError:
     __version__ = ""
 __author__ = "Nathan Taggart"
 __credits__ = "meisnate12"

--- a/arrapi/__init__.py
+++ b/arrapi/__init__.py
@@ -9,7 +9,7 @@ from .apis.lidarr import LidarrAPI
 from .apis.readarr import ReadarrAPI
 
 try:
-    __version__ =  version("arrapi")
+    __version__ = version("arrapi")
 except importlib.metadata.PackageNotFoundError::
     __version__ = ""
 __author__ = "Nathan Taggart"

